src/anemoi/graphs/nodes/builder.py

```diff
@@ -160,13 +158,11 @@ class NPZFileNodes(BaseNodeBuilder):
 
     Methods
     -------
-    get_coordinates()
```

Comment:
This is missing here, see above.

Reply:
The get_coordinates() method already appears in the [Modules section](https://anemoi-graphs--18.org.readthedocs.build/en/18/modules/node_builder.html#anemoi.graphs.nodes.builder.ZarrDatasetNodes.get_coordinates) because the method is implemented in the class. If we include it, then it will appear two times in the documentation